packages/editor/src/functions/EditorControlFunctions.ts

```diff
@@ -283,6 +284,9 @@ const createObjectFromSceneElement = (
 
   EditorState.markModifiedScene(gltfEntity)
 
+  //force hierarry tree state refresh
+  HierarchyTreeState.refresh()
```

Comment:
This is definitely the wrong approach. The hierarchy tree needs to react to the correct data, or we will end up with many to many dependency relationships and the complexity of these functions will grow.

The hierarchy tree needs to be hooked into appropriate data.